src/main/java/uk/gov/justice/hmpps/prison/api/model/OffenderKeyDates.java

```diff
@@ -26,4 +26,10 @@
 
     @ApiModelProperty(value = "SED - date on which sentence expires.", example = "2020-02-03")
     private LocalDate sentenceExpiryDate;
+
+    @ApiModelProperty(required = true, value = "Effective sentence end date.", example = "2020-02-03")
+    private LocalDate effectiveSentenceEndDate;
+
+    @ApiModelProperty(required = true, value = "Sentence length.", example = "11/00/00")
```

Comment:
is this example of sentenceLength right? e.g. "11/00/00".. what does that mean?

Reply:
Note to self - we need to update CRD to include these required fields otherwise the POST will fail once this is merged. Our service is in development yet so not a big deal

Reply:
Updated the SwaggerDoc so hopefully the format is clearer